x11-packages/rnote/build.sh

```diff
@@ -0,0 +1,67 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/flxzt/rnote"
+TERMUX_PKG_DESCRIPTION="An infinite canvas vector-based drawing application for handwritten notes"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Since there are quite extensive patches included with this package please consider setting yourself as the maintainer.
```suggestion
TERMUX_PKG_MAINTAINER="TERMUX_PKG_MAINTAINER="<Name or Pseudonym> @EDLLT"
```